packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -245,10 +245,7 @@ export function BlockEditor( {
 
 	const productFormTemplate = useMemo(
 		function pickAndParseTheProductFormTemplate() {
-			if (
-				! isProductEditorTemplateSystemEnabled ||
-				! selectedProductFormId
-			) {
+			if ( ! isProductFormTemplateEnabled || ! selectedProductFormId ) {
```

Comment:
I think that the function needs to be called, right? `isProductFormTemplateEnabled()`